src/startpage.cpp

```diff
@@ -118,3 +117,7 @@ void StartPage::updateBackground()
     }
     m_background.setDevicePixelRatio(devicePixelRatioF());
 }
+
+void StartPage::on_pathsAndArchSettingsButton_clicked() {
+    emit pathsAndArchSettingsButtonClicked();
+}
```

Comment:
this isn't connected anywhere, is it? generally I'd prefer if you remove this and just connect the signal of the button to the signal of the StartPage directly